landoapi/transplants.py

```diff
@@ -878,7 +881,11 @@ def blocker_prevent_symlinks(
     diff_id = PhabricatorClient.expect(diff, "id")
     parsed_diff = stack_state.parsed_diffs[diff_id]
 
-    return DiffAssessor(parsed_diff=parsed_diff).check_prevent_symlinks()
+    symlink_check = PreventSymlinksCheck()
+    for diff in parsed_diff:
+        symlink_check.next_diff(diff)
+
+    return symlink_check.result()
```

Comment:
I think we should keep using the DiffAssessor here, even if we are making it run a single check. This allows us to keep control of the `for` loop and the `return` in a single location, rather than duplicate it.